jobs/build-podman.Jenkinsfile

```diff
@@ -0,0 +1,379 @@
+                                 --git-url ${params.PODMAN_MACHINE_GIT_URL} \
+                                 --git-sub-dir podman-image-daily \
+                                 --repo ${params.CONTAINER_REGISTRY_STAGING_REPO} \
+                                 --push-to-registry --auth=\$REGISTRY_SECRET
+                         """)
+                     }
+                 }]}
+            }
+            // Initialize the sessions on the remote builders
+            stage("Initialize Remotes") {
+                parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                    if (arch != "x86_64") {
+                        pipeutils.withPodmanRemoteArchBuilder(arch: arch) {
+                            archinfo[arch]['session'] = pipeutils.makeCosaRemoteSession(
+                                // The cache size for aarch64 in runvm is insufficient,
+                                // even when considering that all artifacts are around
+                                // the same size as those for x86_64
+                                env: ["OSBUILD_EXPORT_FORCE_NO_PRESERVE_OWNER":"true", "RUNVM_CACHE_SIZE": "25G"],
+                                expiration: "60m",
+                                image: params.COREOS_ASSEMBLER_IMAGE,
+                                workdir: WORKSPACE,
+                            )
+                            withEnv(["COREOS_ASSEMBLER_REMOTE_SESSION=${archinfo[arch]['session']}"]) {
+                                 shwrap("""cosa init --force https://github.com/coreos/fedora-coreos-config.git""")
+                            }
+                        }
+                    } else {
+                         shwrap("""cosa init --force https://github.com/coreos/fedora-coreos-config.git""")
+                    }
+                }]}
+            }
+
+            stage("Build Disk Images") {
+                  parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                        if (arch != "x86_64") {
+		            pipeutils.withOptionalExistingCosaRemoteSession(
+                                arch: arch, session: archinfo[arch]['session']) {
+                                generate_disk_images(artifacts, shortcommit, arch, params.CONTAINER_REGISTRY_STAGING_REPO)
+		            }
+                        } else {
+                            generate_disk_images(artifacts, shortcommit, arch, params.CONTAINER_REGISTRY_STAGING_REPO)
+                        }
+                   }]}
+	    }
+
+           stage("Kola") {
+               parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                   if (arch != "x86_64") {
+                       pipeutils.withOptionalExistingCosaRemoteSession(
+                           arch: arch, session: archinfo[arch]['session']) {
+                           shwrap("""cosa kola run  --basic-qemu-scenarios --qemu-image=./podman-${shortcommit}.qcow2""")
```

Comment:
Aside: keep an eye on https://github.com/coreos/coreos-assembler/pull/3652; we'll have to update this once that PR merges.